elide-contrib/elide-dynamic-config-helpers/src/main/java/com/yahoo/elide/contrib/dynamicconfighelpers/DynamicConfigHelpers.java

```diff
@@ -121,8 +135,17 @@ public static ElideTableConfig getElideTablePojo(String basePath, Map<String, Ob
      */
     public static ElideTableConfig stringToElideTablePojo(String content, Map<String, Object> variables)
             throws IOException {
+        ElideTableConfig table = null;
         String jsonConfig = hjsonToJson(resolveVariables(content, variables));
-        return getModelPojo(jsonConfig, ElideTableConfig.class);
+        try {
+            if (DynamicConfigSchemaValidator.verifySchema(TABLE, jsonConfig)) {
+                table = getModelPojo(jsonConfig, ElideTableConfig.class);
+            }
+        } catch (ProcessingException e) {
+            log.error("Error Validating Table config : " + e.getMessage());
+            throw new IOException(e);
+        }
+        return (table == null ? new ElideTableConfig() : table);
```

Comment:
If you initialize table to new ElideTableConfig, you can remove this check for null at the end.